src/components/TableDialog.vue

```diff
@@ -139,6 +185,22 @@ export default defineComponent({
     },
   },
   setup(props, { emit }) {
+    // Stepper control
+    const step: Ref<number> = ref(1);
+    const rowSample: Ref<Array<{}>> = ref([]);
```

Comment:
It's a minor thing, but I think a better name for this is `sampleRows`. I feel like that better conveys that this is just meant to be a sample of rows from the whole file. A comment may be good too.

Feel free to push back if you think it's too minor of a change.

Reply:
See 137f1671e635636d08d00e3b0da614191b96cc1d.